src/tests/Common/dirs.proj

```diff
@@ -15,6 +15,8 @@
       <DisabledProjects Include="$(TestRoot)Performance\Scenario\JitBench\unofficial_dotnet\JitBench.csproj" /> <!-- no official build support for SDK-style netcoreapp2.0 projects -->
       <DisabledProjects Include="$(TestRoot)TestWrappers*\**\*.csproj" />
       <DisabledProjects Include="$(TestRoot)nativeaot\**\*.csproj" Condition="'$(TestBuildMode)' != 'nativeaot'" />
+      <!-- Projects can't be build for mobile platforms: No entrypoint module -->
```

Comment:
We don't do target-specific exclusion like this. Could you make it local to the test? Add `CLRTestTargetUnsupported` with `Condition` to the csproj file.